plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/ViperPoweredDeclarationChecker.kt

```diff
@@ -41,19 +42,21 @@ private fun TargetsSelection.applicable(declaration: FirSimpleFunction): Boolean
 
 class ViperPoweredDeclarationChecker(private val session: FirSession, private val config: PluginConfiguration) :
     FirSimpleFunctionChecker(MppCheckerKind.Common) {
+    private val purityChecker = PurityChecker()
+
```

Comment:
This class is only meant for getting everything up and running; all actual purity checking should happen within the `ProgramConverter`.